src/ORM/Query.php

```diff
@@ -1275,4 +1275,22 @@ protected function _decorateResults($result)
 
         return $result;
     }
+
+    /**
+     * Selects all fields in the schema with the exception of the fields you pass
+     * in the $excludedFields parameter.
+     *
+     * @param array $excludedFields fields you don't want to select
+     * @param bool $overwrite whether to reset fields with passed list or not
+     * @return $this
+     */
+    public function selectAllExcept(array $excludedFields = [], $overwrite = false)
+    {
+        $allFields = $this->repository()->getSchema()->columns();
```

Comment:
See response above......

If the developer passes an Association or Table as second parameter, we use that param, otherwise we fallback on ```$this->respository()```